src/schema/contentful/phoenix.js

```diff
@@ -76,6 +78,24 @@ const typeDefs = gql`
     ${entryFields}
   }
 
+  type PersonBlock implements Block {
+    "Name of the person displayed on the block"
+    name: String
+    "The person's position within the organization"
+    type: String
+    "The person's relationship with the organization: active? non-active?"
```

Comment:
And then we can change this one to:
```suggestion
    "The status of the person's relationship with the organization: active? non-active?"
```